packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -190,7 +190,18 @@ Future<void> _runBuild(
 
   // MSBuild sends all output to stdout, including build errors. This surfaces
   // known error patterns.
-  final RegExp errorMatcher = RegExp(r':\s*(?:warning|(?:fatal )?error).*?:');
+  final List<String> errorMessages = <String>[
+    r':\s*(?:warning|(?:fatal )?error).*?:',
+    r'Error detected in pubspec.yaml:',
+  ];
+  // Known secondary error lines for pubspec.yaml errors:
+  final List<String> pubspecErrorDetails = <String>[
+    r'No file or variants found for asset:',
+  ];
+  final RegExp errorMatcher = RegExp(<String>[
+    ...errorMessages,
+    ...pubspecErrorDetails,
+  ].join('|'));
```

Comment:
Does this work? I would expect that each regex would be wrapped in `(`. Something like:

```regex
:(\s*(?:warning|(?:fatal )?error).*?:)|(Error detected in pubspec.yaml:)|(No file or variants found for asset:)
```

Reply:
Since OR in first regex is already inside parantesis and other two regex's dont use OR, it works.
Demo: https://dartpad.dev/db72fced323edcf59414a373f42d49ce

Reply:
Ah interesting!